tests/integration/web/alertprofiles_test.py

```diff
@@ -59,568 +48,589 @@ def test_alertprofiles_view(client, view):
-        type=sms,
-        address=valid_phone_number,
-    ).exists()
-    assert f"Saved address {valid_phone_number}" in smart_str(response.content)
-
-
-def test_alertprofiles_add_valid_phone_number_with_double_zero_country_code_should_succeed(
-    client,
-):
-    """
-    Tests that a valid phone number with a country code with double zero (00xx) can be
-    added"""
-    valid_phone_number = "004747474747"
-    sms = AlertSender.objects.get(name=AlertSender.SMS)
-    url = reverse("alertprofiles-address-save")
-    data = {
-        "address": valid_phone_number,
-        "type": sms.pk,
-    }
-    response = client.post(url, data=data, follow=True)
-    assert response.status_code == 200
-    assert AlertAddress.objects.filter(
-        type=sms,
-        address=valid_phone_number,
-    ).exists()
-    assert f"Saved address {valid_phone_number}" in smart_str(response.content)
-
-
-def test_alertprofiles_add_invalid_phone_number_should_fail(client):
-    """Tests that an invalid phone number cannot be added"""
-    invalid_phone_number = "abc"
-    sms = AlertSender.objects.get(name=AlertSender.SMS)
-    url = reverse("alertprofiles-address-save")
-    data = {
-        "address": invalid_phone_number,
-        "type": sms.pk,
-    }
-    response = client.post(url, data=data, follow=True)
-    assert response.status_code == 200
-    assert not AlertAddress.objects.filter(
-        type=sms,
-        address=invalid_phone_number,
-    ).exists()
-    assert "Not a valid phone number." in smart_str(response.content)
-
-
-def test_alertprofiles_confirm_remove_filter_group(db, client, dummy_filter_group):
-    url = reverse('alertprofiles-filter_groups-remove')
-    response = client.post(
-        url,
-        follow=True,
-        data={
-            'confirm': '1',
-            'element': [dummy_filter_group.id],
-        },
-    )
-    assert response.status_code == 200
-    assert not FilterGroup.objects.filter(pk=dummy_filter_group.pk).exists()
-
-
-def test_alertprofiles_remove_filter_group(db, client, dummy_filter_group):
-    url = reverse('alertprofiles-filter_groups-remove')
-    response = client.post(
-        url,
-        follow=True,
-        data={
-            'filter_group': [dummy_filter_group.id],
-        },
-    )
-    assert response.status_code == 200
-    assert "Confirm deletion" in smart_str(response.content)
-    assert dummy_filter_group.name in smart_str(response.content)
-    assert FilterGroup.objects.filter(pk=dummy_filter_group.pk).count() == 1
+class TestsAlertProfiles:
+    def test_profile_with_nonascii_name_should_be_saved(self, db, admin_account):
+        factory = RequestFactory()
+        request = factory.get(reverse('alertprofiles-profile-save'))
+        request.account = admin_account
+        request.session = MagicMock()
+        profile = AlertProfile(account=request.account, name=u'ÆØÅ')
+        profile.save()
+
+        assert set_active_profile(request, profile) is None
+
```

Comment:
Ugh, I wish diff was smarter. I guess if it had been added to the end of TestAlertProfiles, diff wouldn't have reported as many changed lines.